tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/telemetryMetrics/util/MetricComparator.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.telemetryMetrics.util;
+
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.lessThan;
+
+import org.eclipse.microprofile.fault.tolerance.tck.util.TCKConfig;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+
+public class MetricComparator {
```

Comment:
Although it's called `MetricComparator`, it doesn't actually have anything to do with metrics and doesn't reference either the telemetry or MP Metrics APIs so we can move it to the utils package and avoid duplicating it.